/**:
  ros__parameters:
    origin.latitude: 40.246562
    origin.longitude: -111.647649
    origin.altitude: 1486.0

    water_salinity_ppt: 0.0
    fluid_pressure_atm: 87250.0
    holoocean_vehicle: "auv0"
    frost_vehicle: "coug2"


# depth_reverse:
#   ros__parameters:


ucommand_bridge:
  ros__parameters:
    fin_scalar: -3.0
    publish_thruster: false